### PR TITLE
Docv2/misc

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/function/fuzzy.hpp [hunk 1]
```diff
@@ -26,6 +26,96 @@ namespace eve
     }
   };
 
+  //================================================================================================
```
Comment (jfalcou) [inline include/eve/function/fuzzy.hpp:29]: Split over each object

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: include/eve/concept/value.hpp [hunk 2]
```diff
@@ -28,6 +28,19 @@ namespace eve
   //! - `eve::logical<eve::wide<char>>`
   //! - `float`
   //! - `eve::wide<int, eve::fixed<1>>`
+  //!
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/concept/value.hpp]: done

File: include/eve/detail/function/broadcast.hpp [hunk 3]
```diff
@@ -40,12 +40,13 @@ namespace eve
   //!
   //! **Parameters**
   //!
-  //! `x`:  An instance of eve::simd_value
+  //! `x`:  An instance of an \ref simd_value
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/detail/function/broadcast.hpp:43]: done

File: include/eve/detail/function/broadcast.hpp [hunk 4]
```diff
@@ -40,12 +40,13 @@ namespace eve
...
+  //!
   //! `i`:  An eve::index indicating which lane of `x` to broadcast
   //!
   //! **Return value**
   //!
-  //! If `x` is an instance of a eve::simd_value`T`, the return value is equivalent to `T{x.get(I)}`.
+  //! If `x` is an instance of a \ref simd_value `T`, the call is equivalent to `T{x.get(I)}`.
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/detail/function/broadcast.hpp]: done

File: include/eve/function/acos.hpp [hunk 5]
```diff
@@ -34,11 +34,11 @@ namespace eve
   //!
   //! **Parameters**
   //!
-  //!`x`:   [floating real value](../../concepts.html#value).
+  //!`x`:   [floating real value](@ref eve::value).
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/function/acos.hpp]: done

File: include/eve/memory/is_aligned.hpp [hunk 6]
```diff
@@ -52,9 +52,9 @@ namespace eve
   //! @brief Checks if a pointer satisfy an alignment constraint.
   //!
   //! @param  ptr   Pointer to check
-  //! @param  lanes Alignment constraint to verify expressed as a SIMD register number of lanes.
+  //! @tparam  Lanes Alignment constraint to verify expressed as a SIMD register number of lanes.
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/memory/is_aligned.hpp]: restored

File: include/eve/function/logical_and.hpp [hunk 7]
```diff
@@ -36,15 +36,15 @@ namespace eve
...
+  //!`x`, `y`:   [values](@ref eve::value).
   //!
   //! **Return value**
   //!
   //!Computes  logical AND of the two parameters following the
   //![logical operations semantic](../../../glossary.html#Logical).
   //!
   //!the call `logical_and(x, y)` is semantically equivalent to `x && y`
-  //!if `x` or  `y` is an  [simd value](../../concepts.html#value) and does not shortcut.
+  //!if `x` or  `y` is an  [simd value](@ref eve::value) and does not shortcut.
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/function/logical_and.hpp]: done

File: include/eve/function/bit_shl.hpp [hunk 8]
```diff
@@ -35,19 +35,20 @@ namespace eve
...
+  //!`n`:   An instance of an [integral_value](@ref eve::value).
   //!
   //! **Return value**
   //!
-  //!Computes the [element-wise](../../../glossary.html#element-wise) logical left shift of the first parameter by the second one.
+  //!Computes the [element-wise](@ref glossary_elementwise) logical left shift of the first parameter
+  //!by the second one.
   //!
-  //!the call `bit_shl(x, n)` is equivalent to `x << n` if `x`  is an  [simd value](../../concepts.html#value).
+  //!the call `bit_shl(x, n)` is equivalent to `x << n` if `x`  is an  \link eve::simd_value simd value\endlink.
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/function/bit_shl.hpp]: replaced

File: include/eve/function/cos.hpp [hunk 9]
```diff
@@ -50,13 +50,14 @@ namespace eve
   //!
   //! #### Supported decorators
   //!
-  //!  As all direct trigonometric functions, `cos` supports the restricted, small, medium and big decorators. Click
+  //!  As all direct trigonometric functions, `cos` supports the
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/function/cos.hpp]: done

File: include/eve/function/derivative.hpp [hunk 10]
```diff
@@ -30,4 +30,45 @@ namespace eve
...
+  //================================================================================================
+  //! @addtogroup decorator
+  //! @{
+  //!  @var diff
+  //!
+  //! @brief  Higher-order @callable imbuing derivative semantics onto other @callable{s}.
+  //!
+  //! #### Synopsis
+  //!
+  //!  diff can be declined in
```
[the remark this answers is not in the document]
Comment (jtlap) [reply include/eve/function/derivative.hpp]: done

File: include/eve/function/fuzzy.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/function/fuzzy.hpp:29]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**